### DISCUSSION TITLE
data de aniversariante

### DISCUSSION THREAD

**Discussion by tecdan06** [RamonSilva20/mapos#2031] [General]
seria possivel adicionar um campo de aniversario e mostrar esa data na agenda?
para afim de marketing mandar promoçoes e ou outras coisas.
sera que é possivel?

**Comment by Henrique-Miranda** [comment]
Geralmente em sistemas se coloca a data de nascimento. Seria possivel alterando a estrutura do banco de dados na tabela clientes, incluir um campo nascimento não nulo pois nem todos clientes informam data de nascimento.

**Reply by tecdan06** [reply]
entendi, nao manjo de programaçao, sera que conseguimos colocar no os dados do BD para a agenda?
